pkg/fleet/installer/packages/apminject/apm_inject.go

```diff
@@ -94,7 +95,7 @@ func (a *InjectorInstaller) Setup(ctx context.Context) error {
 	}
 
 	// Create mandatory dirs
-	err = os.Mkdir("/var/log/datadog/dotnet", 0777)
+	err = os.MkdirAll("/var/log/datadog/dotnet", 0755)
```

Comment:
I think in the agent this is created with user `dd-agent`. Can it cause problems if we don't re-own this?